hedera-node/hedera-admin-service-impl/src/main/java/module-info.java

```diff
@@ -5,6 +5,7 @@
     requires com.github.spotbugs.annotations;
     requires dagger;
     requires javax.inject;
+    requires com.hedera.pbj.runtime;
```

Comment:
Are these `requires` statements needed if we don't have any pbj classes in this module?

Reply:
No, they're not strictly needed at this time. I figured they will be needed soon once the freeze service implementation is ported over so I might as well put them in now.

Reply:
I plan anyway to check all the gradle dependencies and module requires in future once we have done the next bigger steps.